src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write your code here
+const headers = Array.from(document.querySelector('tr').children);
+
+const parentHeader = headers[0].parentElement;
+const clonedHeader = headers[1].cloneNode(true);
+
+parentHeader.insertBefore(clonedHeader, headers[4]);
+
+const bodyRows = Array.from(document.querySelector('tbody').children);
+
+for (let i = 0; i < bodyRows.length; i++) {
+  const cells = Array.from(bodyRows[i].children);
+  const clonedCell = cells[1].cloneNode(true);
+
+  bodyRows[i].insertBefore(clonedCell, cells[4]);
```

Comment:
Ensure that `cells[4]` exists before attempting to insert `clonedCell` before it. If `cells` has fewer than 5 elements, this will cause an error.